assets/constants.ts

```diff
@@ -46,6 +46,13 @@ export = {
   // The default language code for an exploration.
   "DEFAULT_LANGUAGE_CODE": "en",
 
+  "NEW_STRUCTURE_TO_COLORS": {
```

Comment:
This naming is confusing. Perhaps ALLOWED_THUMBNAIL_BACKGROUND_COLORS or ALLOWED_THUMBNAIL_BG_COLORS?